source/logrepl/hybrid.go

```diff
@@ -0,0 +1,213 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logrepl
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/jackc/pgx/v4"
+)
+
+const SnapshotInitial = "initial"
+const SnapshotNever = "never"
+
+type Hybrid struct {
+	config Config
+
+	snapshot *SnapshotIterator
+	cdc      *CDCIterator
+
+	conn *pgx.Conn
+}
+
+func NewHybridIterator(ctx context.Context, conn *pgx.Conn, cfg Config) (*Hybrid, error) {
```

Comment:
Hybrid seemed like a better word than `Combined` given how it tightly synchronizes the two iterators instead of just combining their outputs.